doc/STYLE_GUIDE.md

```diff
@@ -16,8 +16,8 @@
 * Avoid personal pronouns in reference documentation ("I", "you", "we").
   * Personal pronouns are acceptable in colloquial documentation such as guides.
   * Use gender-neutral pronouns and gender-neutral plural nouns.
-    * OK: "they", "their", "them", "folks", "people", "developers"
-    * NOT OK: "his", "hers", "him", "her", "guys", "dudes"
+    * OK: "they", "their", "them", "folks", "people", "developers".
```

Comment:
This is not a sentence so I don't think a `.` at the end makes sense here. Same for the line below.